internal/providers/pluginfw/pluginfw_rollout_utils.go

```diff
@@ -0,0 +1,123 @@
+package pluginfw
+
+// This file contains all of the utils for controlling the plugin framework rollout.
+// For migrated resources and data sources, we can add them to the two maps below to have them registered with the plugin framework.
+// Users can manually specify resources and data sources to use SDK V2 instead of the plugin framework by setting the USE_SDK_V2_RESOURCES and USE_SDK_V2_DATA_SOURCES environment variables.
+//
+// Example: USE_SDK_V2_RESOURCES="databricks_library" would force the library resource to use SDK V2 instead of the plugin framework.
+
+import (
+	"os"
+	"strings"
+
+	"github.com/databricks/terraform-provider-databricks/internal/providers/pluginfw/resources/cluster"
+	"github.com/databricks/terraform-provider-databricks/internal/providers/pluginfw/resources/library"
+	"github.com/databricks/terraform-provider-databricks/internal/providers/pluginfw/resources/qualitymonitor"
+	"github.com/databricks/terraform-provider-databricks/internal/providers/pluginfw/resources/registered_model"
+	"github.com/databricks/terraform-provider-databricks/internal/providers/pluginfw/resources/volume"
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+)
+
+// Map of resources that have been migrated from SDK V2 to plugin framework
+var migratedResourceMap = map[string]func() resource.Resource{
+	"databricks_qualitymonitor": qualitymonitor.ResourceQualityMonitor,
```

Comment:
Instead of hard-coding the names here, we can get them by calling the `Metadata()` method for each resource/data source.